tests/unit/fromdate-step-form.spec.ts

```diff
@@ -74,6 +74,21 @@ describe('Convert Date to String Step Form', () => {
     expect(wrapper.find('.customFormat').exists()).toBe(false);
   });
 
+  describe('when user delete content of custom format input', () => {
+    it('should return empty string as format', () => {
+      const wrapper = shallowMount(FromDateStepForm, {
+        store: setupMockStore({}),
+        localVue,
+        propsData: {
+          initialStepValue: { name: 'todate', format: '%Y %m %d %d', column: 'wdc' },
+        },
+      });
+      wrapper.find('.customFormat').vm.$emit('input', undefined); // fake remove input content
```

Comment:
Chipotage : je ne sais pas si c'est la bonne pratique à suivre, mais je préfère quand toute la partie "setup et actions menant à l'état à tester" est dans un `beforeEach` et que les `it` ne contiennent que les assertions :wink:

Reply:
agree, la logique de test était déjà implem comme ça ici :/... Et c'est pas le cas de tout le monde, Eric ne supporte pas les beforeEach par exemple, mais on pourrait en parler en retro front pour essayer de conventionner